css/style.css

```diff
@@ -0,0 +1,270 @@
+* {
+    box-sizing: border-box;
+}
+
+a {
+    text-decoration: none;
+    color: #FAA300;
+}
+
+a:hover {
+    color: #D12411;
+    text-decoration: underline;
+}
+
+nav li {
+    display: inline;
+    padding: 0 35px;
+    font-size: 2rem;
+    font-family: Superclarendon, sans-serif;
+    text-shadow: -1px 0 #000000, 0 1px #000000, 1px 0 #000000, 0 -1px #000000;
```

Comment:
this text shadow and font family declaration is everywhere, just make it its own class and add to all the html elements that need it